### ISSUE TITLE
Pin aggregation

### ISSUE THREAD

**Issue by vonbearshark** [Pitt-CSC/club_project_2017#24]
Should aggregate a collection of pins in the same location into _one_ pin that, when clicked, shows the separate pins living there. Should have a little icon that shows the that there _are_ a number of pins living there. This will help clear up the UI.

**Comment by brlodi** [comment]
Library options seem to be [Leaflet.markercluster](https://github.com/Leaflet/Leaflet.markercluster) and [PruneCluster](https://github.com/SINTEF-9012/PruneCluster). The former seems to be the de facto standard, but the latter seems to look and perform nicer.

**Comment by vonbearshark** [comment]
TIL about PruneCluster. It does look and perform better. And they both need to be imported anyways so it's new overhead regardless. Seems like the best choice. I wonder if we could also prune for all untracked data/data outside Oakland, and set it in a pruned pin fixed on a position on the screen, maybe by dataset type..

**Comment by vonbearshark** [comment]
Don't want to stretch you thin, @brlodi , but any chance you could tackle this, too, since you're working with the pins and logos anyway?

**Comment by brlodi** [comment]
Yeah I can take a look. I found another seemingly more lightweight plugin that merely does the spiderification, so we might consider that too since I don't think the numbered clusters do as good of a job at indicating scale as a tight cluster of pins do (I'd be more in favor if we could tailor the boundaries to some sort of subdivision of Oakland instead of the automatically calculated ones the libraries use, but that seems like a *lot* of work). It resembles the Google Maps implementation of spiderification.

**Comment by vonbearshark** [comment]
What's the plugin?
